ir/src/main/scala/org/scalajs/ir/Trees.scala

```diff
@@ -86,9 +86,13 @@ object Trees {
       ClassIdent(name, None)
   }
 
-  final def isValidJSIdentifier(name: String): Boolean = {
+  /** Tests whether the given name is a valid JavaScript identifier name.
+   *
+   *  This test does *not* exclude keywords.
+   */
+  def isJSIdentifierName(name: String): Boolean = {
     // scalastyle:off return
-    // This method is called on every Name construction; it should be fast.
+    // This method is called on every JSGlobalRef construction; it should be fast.
```

Comment:
It is also called on every `TopLevelExportDef` construction. Maybe find a more general wording?